src/main/webapp/resources/js/components/sequence-files/SequenceFileHeader.jsx

```diff
@@ -18,8 +25,21 @@ export function SequenceFileHeader({
   fileObjectId,
   type,
   removeSampleFiles = () => {},
+  displayConcatenationCheckbox,
 }) {
   const { modifiable } = useSelector((state) => state.sampleReducer);
```

Comment:
You have a lot of branching in here based off of modifiable.  You might want to consider having different componets for each state.

Reply:
Created a separate component if the user can remove or concatenate files for a sample in a0030449ca44d14e86faa4e3ece5b118bf115cc1